### ISSUE TITLE
Restore from existing checkpoint

### ISSUE THREAD

**Issue by christ115** [petuum-inc/poseidon-release#4]
When I use the following code to restore from a pre-trained checkpoint, 
sess.run(init)
saver.restore(sess, tf.train.latest_checkpoint(FLAGS.train_dir))

I found the parameters on each work node are just correct in the first iteration. The training log is as follows,
2017-08-20 23:36:55.186771: step 130001, loss = 1.34, acc = 0.88 (23.6 examples/sec; 1.356 sec/batch)
2017-08-20 23:36:58.913617: step 130002, loss = 12.25, acc = 0.00 (42.5 examples/sec; 0.754 sec/batch)

Looks like the parameters on "ps_server" are not restored properly. Do you have any suggestion to solve this problem?

**Comment by christ115** [comment]
I try to use tf.train.MonitoredTrainingSession instead of "create session, initialize and restore vars manually". Everything works well now!
